### ISSUE TITLE
Multi-platform Docker build

### ISSUE THREAD

**Issue by carlhiggs** [healthysustainablecities/global-indicators#172]
@gboeing suggested we should consider building for multiple architectures - eg. special Docker build (using Dockerx) for ARM as well as AMD64 processors (apparently has advantages for Apple device users).  There are details here https://docs.docker.com/build/building/multi-platform/  

Could be good to do while we're tinkering with the Docker image on the enhancements branch...

**Comment by gboeing** [comment]
The command would be something like:
```
docker buildx build --push --platform=linux/amd64,linux/arm64 -t some_tag .
```

**Comment by carlhiggs** [comment]
Just gave this a go and not sure of exactly why it didn't work, but I suspect something to do with the arm part (as my regular build is linux/amd64).

Here's the command I used
```
docker buildx build --platform linux/amd64,linux/arm64 -t globalhealthyliveablecities/global-indicators:2023-02-06 . --push
```

Anyway, i'll move on and look at this again another time, just wanted to give it a quick go and make a note that not yet working.

**Comment by carlhiggs** [comment]
Just received a request for more explicit ARM64 support (reportedly, processing the example region that takes 8 minutes on a regular not special windows laptop took much longer on an M1 ARM64 MacBook pro -- may partially relate to server timeout addressed in #212, but unusual that that issue should occur for the small example city -- so may be due to build not explicitly supporting ARM64).

Looking further into this, I think more is required than suggested by discussion above, which may be why it didn't work -- ie. its not just changing the build command, according to following link there should be modifications to the dockerfile itself:
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

the example, which has a syntax we're not using referring to an environment variable for `$BUILDPLATFORM`, looks like
```
FROM --platform=$BUILDPLATFORM alpine AS build
# RUN <install build dependecies/compiler>
# COPY <source> .
ARG TARGETPLATFORM
RUN compile --target=$TARGETPLATFORM -o /out/mybinary
```

So --- we currently have a multi stage built we're we take FROM various images; and I think what we should be doing is incorporating the build and target platform variables into this.   Something like that.   I'm looking into this now...

**Comment by carlhiggs** [comment]
The arm64 build still fails, resulting in exit code 127 , which seems to relate to where it tries to run this line:
https://github.com/global-healthy-liveable-cities/global-indicators/blob/27fa54f959b8f719852f122ca9fbb23124ea6ce4/docker/Dockerfile#L48

i don't know why -- am testing this through some tweaks on the enhancements branch that also hope to deal with #193 while we're looking at the docker image
